.travis.yml

```diff
@@ -10,20 +10,20 @@ env:
   global:
   - DOCKER_REPO=$TRAVIS_REPO_SLUG
 before_install:
-- apt-cache madison docker-engine
-- travis_retry sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine
+- apt-cache madison docker-ce
```

Comment:
Though... it's *possible* this is just a breakage in the GitHub UI (I feel like that's happened to me before), the following view from my `git l` alias implies you merged instead of rebased:

```
*   e26c903 - (HEAD -> michaelmoss-master) Merge branch 'master' of https://github.com/michaelmoss/storm (8 hours ago) <Michael Moss>
|\  
| * 1560980 - added comments, call launchDaemon instead of launch (35 hours ago) <Michael Moss>
| * 178d2f1 - #202 (9 days ago) <Michael Moss>
* | f255c4f - added comments, call launchDaemon instead of launch (8 hours ago) <Michael Moss>
* | 96ab89a - #202 (8 hours ago) <Michael Moss>
* |   95837a1 - (origin/master, base/master, master) Merge pull request #205 from JessicaLHartog/log-clarity (11 hours ago) <JessicaLHartog>
|\ \  
| * | da661eb - Fixing non-obvious log line (11 hours ago) <Jessica Hartog>
|/ /  
* |   ca74947 - Merge pull request #207 from erikdw/fix-travis-ci-build (13 hours ago) <Erik Weathers>
|\ \  
| |/  
|/|   
```